modules/core/src/main/java/org/apache/ignite/IgniteSystemProperties.java

```diff
@@ -1113,7 +1114,9 @@
 
     /**
      * WAL rebalance threshold.
+     * @deprecated use Distributed MetaStorage property {@link GridCacheDatabaseSharedManager#HISTORICAL_REBALANCE_THRESHOLD_DMS_KEY}.
```

Comment:
It does not seem correct to provide a link to the internal package/class/field. IgniteSystemProperties is a kind of public API, I think. Perhaps, we need to create a separate class that should contain names of public distributed properties. What do you think?

Reply:
changed to {@code history.rebalance.threshold}